frontend/app/assets/javascripts/spree/frontend/checkout/payment.js

```diff
@@ -4,7 +4,7 @@ var CARD_EXPIRATION_SELECTOR = '.cardExpiry'
 var CARD_CODE_SELECTOR = '.cardCode'
 
 //= require spree/frontend/coupon_manager
-Spree.ready(function ($) {
+window.addEventListener('turbolinks:load', function() {
```

Comment:
Is this okay? To get the translation to pickup I needed to change this.

Reply:
@MatthewKennedy yes, I suppose we could alter `Spree.ready` method - https://github.com/spree/spree/blob/32be7211a6b3510d80c932bcb2f611a8afc28f1a/core/app/assets/javascripts/spree.js#L5

to use `window.addEventListener` rather than `jquery`

Reply:
That works perfectly.